munit/js/src/main/scala/munit/internal/Paths.scala

```diff
@@ -1,10 +1,7 @@
-package java.nio.file
+package munit.internal
```

Comment:
Let's introduce a new `munit.internal.io` package. The general structure I try to following is

- flat list of classes/objects inside `munit._` for public API
- flat list of packages inside `munit.internal._`

The reason is mostly historical because MUnit includes a reimplementation of the java difflib and it didn't feel right to have all of those diff classes inside `munit.internal._`.

Reply:
Done.